libqtile/core/manager.py

```diff
@@ -1031,30 +1031,35 @@ def run_in_executor(self, func: Callable, *args: Any) -> asyncio.Future:
     @expose_command()
     def debug(self) -> None:
         """Set log level to DEBUG"""
+        self.core.update_backend_log_level()
         logger.setLevel(logging.DEBUG)
         logger.debug("Switching to DEBUG threshold")
 
     @expose_command()
     def info(self) -> None:
         """Set log level to INFO"""
+        self.core.update_backend_log_level()
```

Comment:
Don't you want these all after the setLevel() call?

But ideally this wouldn't use side effects, and would take the new log level as an argument so the ordering wouldn't matter.